Assignment2/rober11r-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,36 @@
+{
+	"@context":{
+		"me":"http://upm.com/",
+		"fullName":"me:fullName",
+		"hometown":"me:hometown",
+		"age":"me:age",
+		"almaMater":"me:almaMater",
+		"nameParents":"me:nameParents",
+		"subjects":"me:subjects",
+		"name":"subjects:name",
+		"teachers":"subjects:teachers",
+		"description":"subjects:description"
+	},
+	"@id":"http://upm.com/Roberto/",
+	"@type":"me:Roberto",
```

Comment:
Syntactically is ok, but it is better to define you as a Person or Student :-)
You don't need to change it. Just a minor comment on it.
Good job!